src/components/App/App.tsx

```diff
@@ -7,11 +7,13 @@ import Button from "../Button/Button";
 import CountryFact from "../CountryFact/CountryFact";
 
 const App = () => {
-  // Set states
   const [countriesData, setCountriesData] = useState<any[]>([]);
   const [currentCountry, setCurrentCountry] = useState<string>("");
   const [userScore, setUserScore] = useState<number>(0);
-  const [highScore, setHighScore] = useState<number>(10);
+  const [highScore, setHighScore] = useState<number>(() => {
```

Comment:
I've never seen a callback function set as the default state before, is this best practice?

Could it be refactored to add another state with `[savedHighScore, setSavedHighScore] = useState<number>()`?

That way you might not have to use a callback function as the default state argument.